src/pybind/mgr/ansible/run-tox.sh

```diff
@@ -29,7 +29,9 @@ source ${MGR_ANSIBLE_VIRTUALENV}/bin/activate
 if [ "$WITH_PYTHON2" = "ON" ]; then
   ENV_LIST+="py27,"
 fi
-if [ "$WITH_PYTHON3" = "3" ]; then
+# WITH_PYTHON3 might be set to "ON" or to the python3 RPM version number
+# prevailing on the system - e.g. "3", "36"
+if [ $(perl -e '"'"$WITH_PYTHON3"'" =~ m/(^3|^ON)/; print "$1\n";') ] ; then
```

Comment:
i'd suggest use the builtin regexp support in bash
```sh
if [[ "$WITH_PYTHON3" =~ (ON|3.*) ]]; then
```

Reply:
good catch! done